ci/tests/general/general_vdisk.py

```diff
@@ -71,6 +71,7 @@ def create_volume(size, vpool, name=None, loop_device=None, root_client=None, wa
             if loop_device is not None:
                 root_client.run('umount /mnt/{0} | echo true'.format(loop_device))
                 root_client.run('losetup -d /dev/{0} | echo true'.format(loop_device))
+                root_client.run('rm {0} | echo true'.format(location))
```

Comment:
Do you really want this?
Since each test should be able to run indepedently i'd expect a previous test should clean up its created files